### PR TITLE
Clearing property value types [Step 9] 

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

**Review by Eism — APPROVED** [review]

File: src/engraving/property/propertyvalue.cpp [hunk 1]
```diff
@@ -216,9 +225,14 @@ QVariant PropertyValue::toQVariant() const
...
 
-    // time
+    // Duration
     case P_TYPE::FRACTION:    return QVariant::fromValue(value<Fraction>().toString());
     case P_TYPE::TDURATION:   return QVariant::fromValue(toTDuration());
+
+    // Types
+    case P_TYPE::LAYOUTBREAK_TYPE: return static_cast<int>(value<LayoutBreakType>());
+    case P_TYPE::VELO_TYPE:        return static_cast<int>(value<VeloType>());
+
```
Comment (Jojo-Schmitz) [inline src/engraving/property/propertyvalue.cpp:235]: There are some more "Types" below

File: src/engraving/property/propertyvalue.cpp [hunk 2]
```diff
@@ -267,12 +281,17 @@ PropertyValue PropertyValue::fromQVariant(const QVariant& v, P_TYPE type)
...
     case P_TYPE::FRACTION:      return PropertyValue(Fraction::fromString(v.toString()));
     case P_TYPE::TDURATION: {
         UNREACHABLE; //! TODO
     }
     break;
+
+    // Types
+    case P_TYPE::LAYOUTBREAK_TYPE: return PropertyValue(LayoutBreakType(v.toInt()));
+    case P_TYPE::VELO_TYPE:        return PropertyValue(VeloType(v.toInt()));
+
```
Comment (Jojo-Schmitz) [inline src/engraving/property/propertyvalue.cpp:294]: same

File: src/engraving/property/propertyvalue.cpp (hunk 1 above)
Comment (igorkorsukov) [reply src/engraving/property/propertyvalue.cpp:235]: Step by step, I haven't reached them yet

File: src/engraving/property/propertyvalue.cpp (hunk 2 above)
Comment (igorkorsukov) [reply src/engraving/property/propertyvalue.cpp:294]: same )

**Pull request merged**